pyra/webapp.py

```diff
@@ -100,7 +101,37 @@ def home() -> render_template:
     --------
     >>> home()
     """
-    return render_template('home.html', title='Home')
+    chart_types = hardware.chart_types()
+    chart_translations = hardware.chart_translations
+
+    return render_template('home.html', title='Home', chart_types=chart_types, translations=chart_translations)
+
+
+@app.route('/callback/dashboard', methods=['GET'])
+def callback_dashboard() -> str:
+    """
+    Get dashboard data.
+
+    This should be used in a callback in order to update charts in the web app.
+
+    Returns
+    -------
+    str
+        A list, formatted as a string, containing dictionaries.
+        Each dictionary is a chart ready to use with ``plotly``.
+
+    See Also
+    --------
+    pyra.hardware.chart_data : Returns the same data as this function.
+
+    Examples
+    --------
+    >>> callback_dashboard()
+    '[{"data": [...], "layout": ..., "config": ..., {"data": ...]'
+    """
+    data = hardware.chart_data()
+
+    return data
```

Comment:
Why aren't you using flask.jsonify here? Is this implicit? if so the docstring should probably mention that it returns json.

Also `return hardware.chart_data()` would be better probably.

Reply:
I think I tried it and it wasn't working. Probably I did something wrong, or maybe it's related using PlotlyJSONEncoder in L391 of hardware.py.

Reply:
Yeah I would move the JSON encoding here as I mentioned elsewhere. That would improve the separation of concerns a bit I think.